services/client/src/components/DocsTab.js

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import { useTranslation } from 'react-i18next';
+import { Button } from 'antd';
+
+import MarkdownViewer from 'components/MarkdownViewer';
+import Loader from 'components/Loader';
+
+import useVersionDocs from '../hooks/useVersionDocs';
+
+import downloadBlob from '../utils/downloadBlob';
+
+import s from './DocsTab.module.css';
+
+const DocsTab = ({ versionId, toolbar = false }) => {
+  const { t } = useTranslation();
+
+  const [docData] = useVersionDocs({ versionId });
+  const markdown = docData?.data?.versions_by_pk?.markdown_doc;
+
+  const downloadDoc = useCallback(() => {
+    const blob = new Blob([markdown.replace(/&nbsp;/g, '')]);
+    downloadBlob(blob, `${versionId}.md`);
+  }, [markdown, versionId]);
+
+  if (docData.loading) {
+    return <Loader spinning />;
+  }
+
+  return (
+    <>
+      { toolbar && (
```

Comment:
let's place `(docs)` link near the branch title, keep docs as a tab it not helpful, in my opinion